src/extension/options-page/DashboardRouters/Wallets.tsx

```diff
@@ -85,6 +85,31 @@ export default function DashboardWalletsRouter() {
 
     const tokens = useTokens(current)
     const [detailedTokens, detailedTokensCallback] = useTokensDetailedCallback(tokens)
+    const rightIcons = [
+        <IconButton
+            onClick={() => {
+                if (!currentWallet) return
+                openWalletHistory({
+                    wallet: currentWallet,
+                    onRedPacketClicked(payload) {
+                        openWalletRedPacketDetail({
+                            wallet: currentWallet,
+                            payload,
+                        })
+                    },
+                })
+            }}>
+            <RestoreIcon />
+        </IconButton>,
+        <IconButton
+            onClick={() => {
+                if (currentWallet) openAddToken({ wallet: currentWallet })
+                else openWalletCreate()
+            }}>
+            <AddIcon />
+        </IconButton>,
+    ]
+    !currentWallet && rightIcons.shift()
```

Comment:
How about simplifying the logic like:
```
let rightIcons = [
<IconButton
            onClick={() => {
                if (currentWallet) openAddToken({ wallet: currentWallet })
                else openWalletCreate()
            }}>
            <AddIcon />
        </IconButton>,
]
if (currentWallet) {
    rightIcons.insert(THE_RESTORE_ICON, at: 0)
}
```
`rightIcons.shift` is bug-prone and confusing for others to debug & maintain

Reply:
okay